api/src/main/java/org/eclipse/microprofile/openapi/models/media/Schema.java

```diff
@@ -926,29 +926,33 @@ default Schema writeOnly(Boolean writeOnly) {
     }
 
     /**
-     * Returns the example property from this Schema instance.
+     * Returns the first value from the examples property from this Schema instance.
      *
      * @return an object which is an example of an instance of this Schema
-     **/
+     * @deprecated use {@link #getExamples()}
+     */
+    @Deprecated(since = "4.0")
     Object getExample();
```

Comment:
`example` is still a valid property [in the OpenAPI spec](https://spec.openapis.org/oas/v3.1.0.html#fixed-fields-19) so I should probably change this back.

Reply:
Fixed this.